app/parse.py

```diff
@@ -15,8 +22,90 @@ class Product:
     num_of_reviews: int
 
 
+def get_soup(url):
+    content = requests.get(url).content
+    soup = BeautifulSoup(content, "html.parser")
+    return soup
+
+
+def get_data_from_soup(soup: BeautifulSoup, url: str):
+    if soup.select_one(".btn"):
+        driver = webdriver.Chrome()
+        driver.get(url)
+        actions = ActionChains(driver)
+        try:
+            while True:
+                button = driver.find_element(By.CSS_SELECTOR,
+                                             "a.btn.btn-lg.btn-block.btn-primary.ecomerce-items-scroll-more")
+                driver.execute_script("window.scrollTo(0, document.body.scrollHeight);")
+                actions.move_to_element(button).click().perform()
```

Comment:
The `while True` loop can cause an infinite loop if the button is always present. Consider adding a condition to break out of the loop when no more products are available or after a certain number of iterations to prevent the program from hanging indefinitely.